tests_e2e/test_selenium.py

```diff
@@ -0,0 +1,36 @@
+import os
+from threading import Thread
+
+import pytest
+from selenium import webdriver
+
+import api.trello_helper as trello
+import app
+
+@pytest.fixture(scope="module")
+def driver():
+    with webdriver.Firefox() as driver:
+        yield driver
+
+@pytest.fixture(scope='module')
+def test_app():
+    # Create the new board & update the board id environment variable
+    board_id = trello.add_board()
+    os.environ['TRELLO_BOARD_ID'] = board_id
```

Comment:
Are you sure this is updating the config value?

env_vars.py is defining variables when it gets imported. You are then updating os.environ after that. env_vars.BOARD_ID will still be the original. 

You could make them functions that get it from os.environ or nicer would be to set up the config from os.environ when you create the app.